mne/utils/linalg.py

```diff
@@ -156,3 +156,19 @@ def sqrtm_sym(A, rcond=1e-7, inv=False):
         use_s = s
     a = np.matmul(u * use_s[..., np.newaxis, :], u.swapaxes(-2, -1))
     return a, s
+
```

Comment:
@larsoner you asked me to also check `sqrtm_sym` (above). Is it for Hermitian matrices? If so, it should be fine?

Reply:
it's only for real matrices so it should be fine